test/e2e/aks_fleets_member.go

```diff
@@ -138,11 +138,11 @@ func AKSFleetsMemberSpec(ctx context.Context, inputGetter func() AKSFleetsMember
 		g.Expect(aks.ManagedCluster.Properties.ProvisioningState).NotTo(Equal(ptr.To("Updating")))
 	}, input.WaitIntervals...).Should(Succeed())
 
-	By("Deleting the fleets member")
-	fleetsMemberPoller, err := fleetsMemberClient.BeginDelete(ctx, groupName, fleetName, input.Cluster.Name, nil)
-	Expect(err).NotTo(HaveOccurred())
-	_, err = fleetsMemberPoller.PollUntilDone(ctx, nil)
-	Expect(err).NotTo(HaveOccurred())
+	By("Waiting for the FleetsMember to be deleted")
```

Comment:
This should fulfill e2e coverage of this new functionality. And also the securitygroups test once we transition that to ASO.